components/automate-cli/cmd/chef-automate/initConfigHaTmpl.go

```diff
@@ -77,6 +77,12 @@ enable_custom_certs = false
 [chef_server.config]
 ## === INPUT NEEDED ===
 
+# Chefserver Load Balancer FQDN eg.: "chefserver.example.com"
```

Comment:
I think this also needs to be added to Docs also in Sample Config and other places

Reply:
will raise separate PR for this